qa/c8-orchestration-cluster-e2e-test-suite/tests/tasklist/task-details.spec.ts

```diff
@@ -489,44 +489,44 @@ test.describe('task details page', () => {
     await expect(taskDetailsPage.checkbox).toBeChecked();
   });
 
-  test('task completion with select form', async ({
-    taskPanelPage,
-    taskDetailsPage,
-  }) => {
-    await taskPanelPage.filterBy('Unassigned');
-    await taskPanelPage.openTask('Select User Task');
-    await taskDetailsPage.clickAssignToMeButton();
-
-    await expect(taskDetailsPage.unassignButton).toBeVisible();
-    await taskDetailsPage.selectDropdownValue('Value');
-    await taskDetailsPage.clickCompleteTaskButton();
-    await expect(taskDetailsPage.taskCompletedBanner).toBeVisible();
-
-    await taskPanelPage.filterBy('Completed');
-    await taskPanelPage.assertCompletedHeadingVisible();
-    await taskPanelPage.openTask('Select User Task');
-
-    await expect(taskDetailsPage.form).toContainText('Value');
-  });
-
-  test('task completion with radio button form', async ({
-    taskPanelPage,
-    taskDetailsPage,
-  }) => {
-    await taskPanelPage.filterBy('Unassigned');
-    await taskPanelPage.openTask('Radio Button Task');
-    await taskDetailsPage.clickAssignToMeButton();
-
-    await taskDetailsPage.clickRadioButton('Value');
-    await taskDetailsPage.clickCompleteTaskButton();
-    await expect(taskDetailsPage.taskCompletedBanner).toBeVisible();
-
-    await taskPanelPage.filterBy('Completed');
-    await taskPanelPage.assertCompletedHeadingVisible();
-    await taskPanelPage.openTask('Radio Button Task');
-
-    await taskDetailsPage.assertItemChecked('Value');
-  });
+  test.fixme(
```

Comment:
These tests are temporarily skipped because of [this](https://github.com/camunda/camunda/pull/37438#discussion_r2336647719)